ts/Stock/Indicators/MultipleLinesComposition.ts

```diff
@@ -167,6 +198,46 @@ namespace MultipleLinesComposition {
         return IndicatorClass as (T&typeof Composition);
     }
 
+
+    /**
+     * function to create the path based on the points
+     * @param this Indicator
+     * @param points Points on which the path should be created
+     */
+    function getGraphPath(this: Composition, points: Array<LinePoint>): SVGPath {
+        let indicator = this,
+            path: SVGPath = [],
+            spanA: SVGPath,
+            spanAarr: SVGPath = [];
+
+        points = points || this.points;
+
+        // Render Span
+        if (indicator.fillGraph && indicator.nextPoints) {
+            spanA = SeriesRegistry.seriesTypes.sma.prototype.getGraphPath.call(
```

Comment:
Here (and in some other places), you can use already defined `SMAIndicator.prototype` instead of `SeriesRegistry.seriesTypes.sma`